server.js

```diff
@@ -20,15 +15,70 @@ mongoose.Promise = Promise;
 const port = process.env.PORT || 8080;
 const app = express();
 
+const Stream = mongoose.model("Stream", {
+  show_id: Number,
+  title: String,
+  director: String,
+  cast: String,
+  country: String,
+  date_added: String,
+  release_year: Number,
+  rating: String,
+  duration: String,
+  listed_in: String,
+  description: String,
+  type: String
+})
+
+
+if(process.env.RESET_DB) {
+  const seedDatabase = async () => {
+    await Stream.deleteMany()
+    netflixData.forEach( singleStream => {
+      const NewStream = new Stream(singleStream)
+      NewStream.save()
+    })
+  }
+  seedDatabase()
+ }
+
 // Add middlewares to enable cors and json body parsing
 app.use(cors());
 app.use(express.json());
 
+// When mongoDB is not active
+app.use((req, res, next) => {
+  if (mongoose.connection.readyState === 1) {
+    next()
+  } else {
+    res.status(503).json({ error: "Service unavailable" })
+  }
+})
+
 // Start defining your routes here
 app.get("/", (req, res) => {
-  res.send("Hello Technigo!");
+  res.send(listEndpoints(app));
```

Comment:
Nice that you are listing all of the endpoints at the home page. Endpoints naming is also good and easily understood :)